__snapshots__/src/index.test.js.md

```diff
@@ -16,8 +16,12 @@ Generated by [AVA](https://ava.li).
         className="dropdown"
       >
         <a
+          'aria-expanded'={false}
+          "aria-haspopup"="listbox"
```

Comment:
Interesting how the renderer emits single quotes for a pro and double quotes for another.

Reply:
Yeah, I've had some problems with snapshots being different depending on what machine I used. My Mac liked to output the markdown header with slash instead of backslash :/ Not sure what I can do about this